src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -200,6 +200,43 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 		const endsWithSpace = cursorPrefix.endsWith(' ');
 		const lastWord = endsWithSpace ? '' : cursorPrefix.split(' ').at(-1) ?? '';
 
+		if (foldersRequested) {
+			if (!lastWord.trim()) {
+				resourceCompletions.push({
+					label: '.',
+					provider: 'builtin',
```

Comment:
Shouldn't this include the actual provider since we want to know what extension(s) are requesting those resources.